sats-dna/src/main/kotlin/com/sats/dna/components/SatsOutlinedTextField.kt

```diff
@@ -86,6 +143,101 @@ fun M3SatsOutlinedTextField(
     )
 }
 
+object M3SatsOutlinedTextFieldDefaults {
+    @Composable
+    fun colors(
+        focusedTextColor: Color = Color.Unspecified,
+        unfocusedTextColor: Color = Color.Unspecified,
+        disabledTextColor: Color = Color.Unspecified,
+        errorTextColor: Color = Color.Unspecified,
+        focusedContainerColor: Color = Color.Unspecified,
+        unfocusedContainerColor: Color = Color.Unspecified,
+        disabledContainerColor: Color = Color.Unspecified,
+        errorContainerColor: Color = Color.Unspecified,
+        cursorColor: Color = Color.Unspecified,
+        errorCursorColor: Color = Color.Unspecified,
+        selectionColors: TextSelectionColors? = null,
+        focusedBorderColor: Color = SatsTheme.colors2.graphicalElements.border.focused,
+        unfocusedBorderColor: Color = SatsTheme.colors2.graphicalElements.border.default,
+        disabledBorderColor: Color = SatsTheme.colors2.graphicalElements.border.default,
+        errorBorderColor: Color = SatsTheme.colors2.graphicalElements.signal.error,
```

Comment:
Our own defaults, can be overridden if needed. Should we allow overriding these?

Reply:
Same as above